lib/IAMClient.d.ts

```diff
@@ -360,6 +360,7 @@ declare class VaultClient {
         logger?: werelogs.RequestLogger;
     }, callback: Function): undefined;
     healthcheck(reqUid: any, callback: any): void;
+    report(reqUid: any, callback: any): void;
```

Comment:
nothing particular on this specific PR, but I am wondering on the name/semantics of the API...Probably to be discussed in the associated Vault PR